docs/features.rst

```diff
@@ -25,37 +23,40 @@ Format
         ],
     }
 
-How it works
-````````````````````````
-
-Internally is this parameter used for constructing `WHERE` clause, with *task_ts* and (*task_ts* - *days*) as
-interval borders. In case multiple columns are named, all would be joined with `AND` - therefore only rows which satisfy
-all conditions at the same time will be eligible for further checking.
-
+    # will select data that has `a` column between running time and running time minus 10 days AND `b` column
+    # between running time and running time minus 24 days
+    # then it will do the check against those filtered data only
 
 Miscellaneous
 ````````````````````````
 
- - For backward-compatibility is also supported simple format, which defines only name of the column. In this case default (30) for *days* is used.
+- For backward-compatibility is also supported simple format, which defines only name of the column. In this case default (30) for *days* is used.
 
 .. code-block:: json
 
     {
         "time_filter": "a"
     }
 
-- Parallel to **columns** shortcut for defining multiple rules at once, parameter **separate_time_filters** is available.
-- Each one will be used separately. In case both *separate_time_filters* and *time_filter* are defined, only *separate_time_filters* would be considered.
+- Parameter **separate_time_filters** is available.
+
+  - Each one will be used separately. Meaning, it would actually create X rules and therefore X checks would be done against the data, each filtering the data by passed column. In case both *separate_time_filters* and *time_filter* are defined, only *separate_time_filters* would be considered.
```

Comment:
This seems to be unclear, did I get it correctly? `Creates a matrix of rules and filters - i.e. provided rule is applied with each separate_time_filters value separately - acts as a generator of rules.`